cypress/e2e/pages/homePage/index.cy.js

```diff
@@ -1,33 +1,87 @@
+import runTestsForPage from '#nextjs/cypress/support/helpers/runTestsForPage';
 import { HOME_PAGE } from '#app/routes/utils/pageTypes';
-import getUrls from './urls';
-import getAppEnv from '../../../support/helpers/getAppEnv';
-import visitPage from '../../../support/helpers/visitPage';
-import crossPlatformTests from './tests';
+import getOptimizelyKey from '../../../support/helpers/getOptimizelyKey';
 import canonicalTests from './testsForCanonicalOnly';
 
-const pageType = HOME_PAGE;
+const tests = [canonicalTests];
 
-const environment = getAppEnv();
-const urls = getUrls();
+const testSuites = [
+  {
+    path: '/arabic',
+    runforEnv: ['local', 'test', 'live'],
+    service: 'arabic',
+    tests,
+  },
+  {
+    path: '/kyrgyz',
+    runforEnv: ['local', 'test', 'live'],
+    service: 'kyrgyz',
+    tests,
+  },
+  {
+    path: '/persian',
+    runforEnv: 'local',
+    service: 'persian',
+    tests,
+  },
+  {
+    path: '/pidgin',
+    runforEnv: ['local', 'test'],
+    service: 'pidgin',
+    tests,
+  },
+  {
+    path: '/polska',
+    runforEnv: ['local', 'test', 'live'],
+    service: 'polska',
+    tests,
+  },
+  {
+    path: '/portuguese',
+    runforEnv: ['local', 'test', 'live'],
+    service: 'portuguese',
+    tests,
+  },
+  {
+    path: '/serbian/lat',
+    runforEnv: ['local', 'test', 'live'],
+    service: '/serbian/lat',
+    tests,
+  },
+  {
+    path: '/serbian/cyr',
+    runforEnv: ['local', 'test', 'live'],
+    service: '/serbian/cyr',
+    tests,
+  },
+  {
+    path: '/uzbek/lat',
+    runforEnv: ['local', 'test', 'live'],
+    service: '/uzbek/lat',
+    tests,
+  },
+  {
+    path: '/uzbek/cyr',
+    runforEnv: ['local', 'test', 'live'],
+    service: '/uzbek/cyr',
+    tests,
+  },
+];
 
-urls.forEach(url => {
-  const { service, variant = 'default' } = url;
-  const currentPath = url[environment];
+describe('Home Page', () => {
+  beforeEach(() => {
+    cy.intercept(
+      {
+        url: `https://cdn.optimizely.com/datafiles/${getOptimizelyKey()}.json`,
+      },
+      request => {
+        request.reply({ statusCode: 404 });
+      },
```

Comment:
I am also wondering if we should include this in all tests - at the moment, we're duplicating this here & in the article / story page tests - @amoore108 what do you think?

Reply:
If there is a way to disable it globally in Cypress that would be good. We do stop Optimizely actually rendering in the `withOptimizelyProvider` in Cypress, but the datafile will still download.

I guess though, is it bad that the datafile downloads in Cypress? Think it was a measure put in place to stop experiments running in Cypress, but the `withOptimizelyProvider` check should stop that anyway?